ms2deepscore/wrapper_functions/training_wrapper_functions.py

```diff
@@ -38,28 +39,28 @@ def train_ms2deepscore_wrapper(spectra_file_path,
                                              split_fraction=validation_split_fraction)
 
     # Split training in pos and neg and create val and training split and select for the right ionisation mode.
-    training_spectra = stored_training_data.load_training_data(settings.ionisation_mode, "training")
-    validation_spectra = stored_training_data.load_training_data(settings.ionisation_mode, "validation")
+    training_spectra = stored_training_data.load_training_data(model_settings.ionisation_mode, "training")
+    validation_spectra = stored_training_data.load_training_data(model_settings.ionisation_mode, "validation")
 
     # Train model
     train_ms2ds_model(training_spectra, validation_spectra,
                       stored_training_data.trained_models_folder,
-                      settings)
+                      model_settings, generator_settings)
 
     # Create performance plots for validation spectra
     ms2deepsore_model_file_name = os.path.join(stored_training_data.trained_models_folder,
-                                               settings.model_directory_name,
-                                               settings.model_file_name)
+                                               model_settings.model_directory_name,
+                                               model_settings.model_file_name)
     calculate_true_values_and_predictions_for_validation_spectra(
```

Comment:
We still have to integrate the use of the datagenerator for validation spectra. It currently works as well, with the implementation I made before: Matching all inchikeys against each other one time, sampling a random spectra for each, repeating a 100 times. But this is also not the exact same over time.

Reply:
But this is a future PR, I will add it to the todo list in dev_pytorch